README.md

```diff
@@ -20,7 +20,25 @@ See the documentation for `zicht/cms` (https://github.com/zicht/cms)
 for more information on how the `zicht/page-bundle` relates to the
 other Zicht Bundles
 
+## Voters
+To enable the use of [voters](http://symfony.com/doc/current/security/voters.html) in your configuration you should properly 
+configure `security.yml` or in your config the `security` part
+
+At least you should have these lines
+```
+security:
+    access_decision_manager:
+            strategy: unanimous
+```
+
+## PageVoter
```

Comment:
This is a part of the Voters section, which is marked up with two hashes, so this should be three hashes.